CONTRIBUTING.md

```diff
@@ -13,4 +13,29 @@ See LICENSE file for more information
 * Whatever content you contribute will be provided under the license(s) associated with the JSONForms project. Whatever content you contribute may also be provided under any open-source license which is or will be approved by the Open Source Initiative (OSI). This specifically includes but is not limited to the following licenses: Apache License 2.0, BSD 2/3 License, MIT License and Eclipse Public License v1.0.
 * You understand and agree that the JSONForms project and your contributions are public, and that a record of the contribution (including all personal information you submit with it, including your sign-off) is maintained indefinitely and may be redistributed consistent with the license(s) involved.
 
-You need to accept our <a href="https://www.clahub.com/agreements/eclipsesource/jsonforms">CLA (Contributer License Agreement)</a> before we can accept your contribution.
\ No newline at end of file
+You need to accept our <a href="https://gist.github.com/koegel/d4623fb46994d9b75502">CLA (Contributor License Agreement)</a> before we can accept your contribution.
+* Accept by CLAHub: <a href="https://www.clahub.com/agreements/eclipsesource/jsonforms">https://www.clahub.com/agreements/eclipsesource/jsonforms</a>.
+* or accept via email. Please send the email at the bottom of this page to jsonforms-cla@eclipsesource.com. Replace the blocks marked with '[]' with your personal information.
+
+Once accepted you will be added to the CONTRIBUTORS.md list.
+
+# Email
+```
+I accept the Contributor License Agreement which can be found at 
+
+https://gist.github.com/koegel/d4623fb46994d9b75502
```

Comment:
I am also fine with putting it in the repository. On Gist you immediately see all revisions and history of the CLA but of course this information is also available in the repository.